bundles/org.openhab.binding.wundergroundupdatereceiver/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,568 @@
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rain-weekly" advanced="true">
+		<item-type>Number:Length</item-type>
+		<label>Weekly Rain</label>
+		<description>Rain since the start of this week.</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rain-monthly" advanced="true">
+		<item-type>Number:Length</item-type>
+		<label>Monthly Rain</label>
+		<description>Rain since the start if this month.</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="rain-yearly" advanced="true">
+		<item-type>Number:Length</item-type>
+		<label>Yearly Rain</label>
+		<description>Rain since the start of this year.</description>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+
+	<channel-type id="metar" advanced="true">
+		<item-type>String</item-type>
+		<label>METAR weather report</label>
+		<description>METAR formatted weather report</description>
+		<category>Sun_Clouds</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="clouds" advanced="true">
+		<item-type>String</item-type>
+		<label>Cloud cover</label>
+		<description>METAR style cloud cover.</description>
+		<category>Sun_Clouds</category>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<!-- for sensors 2,3,4 use soilmoisture2, soilmoisture3, and soilmoisture4 -->
+	<channel-type id="soil-moisture" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Soil Moisture</label>
+		<description>Soil moisture in %.</description>
+		<category>Moisture</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Water</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %%"/>
+	</channel-type>
+
+	<!-- for sensor 2 use leafwetness2 -->
+	<channel-type id="leafwetness" advanced="true">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Leaf wetness</label>
+		<description>Leaf wetness in %.</description>
+		<category>Moisture</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Water</tag>
+		</tags>
+		<state readOnly="true" pattern="%.0f %%"/>
+	</channel-type>
+
+	<channel-type id="solarradiation">
+		<item-type>Number:Intensity</item-type>
+		<label>Solar Radiation</label>
+		<description>Solar radiation</description>
+		<category>Sun</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="uv">
+		<item-type>Number:Dimensionless</item-type>
+		<label>UV index</label>
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

```suggestion
		<label>UV Index</label>
```